package.json

```diff
@@ -76,25 +76,22 @@
   "resolutions": {
     "**/@types/node": ">=10.17.17 <10.20.0",
     "**/cross-fetch/node-fetch": "^2.6.1",
-    "**/dns-packet": "^1.3.4",
     "**/fast-deep-equal": "^3.1.1",
+    "**/front-matter": "^4.0.2",
     "**/graphql-toolkit/lodash": "^4.17.15",
     "**/hoist-non-react-statics": "^3.3.2",
     "**/immer": "^8.0.1",
     "**/isomorphic-fetch/node-fetch": "^2.6.1",
     "**/istanbul-instrumenter-loader/schema-utils": "^1.0.0",
     "**/load-grunt-config/lodash": "^4.17.20",
-    "**/locutus": "^2.0.14",
```

Comment:
> unnecessary resolutions for dependencies 

Just verifying that this was unnecessary https://github.com/opensearch-project/OpenSearch-Dashboards/pull/460

Reply:
Yeah that's correct, the correct way to bump the minor or patch version is to remove the dependency from the lockfile and then re-run `bootstrap`. That's what dependabot does as well: https://medium.com/@ayushya/upgrading-javascript-packages-deep-dependencies-using-yarn-8b5983d5fb6b

Reply:
nice to learn